### ISSUE TITLE
Move dependencies to React 15?

### ISSUE THREAD

**Issue by devang-kredx** [nnarhinen/react-pdf#24]
```
$npm install react-pdf --save
my-application
├── graceful-fs@4.1.3 
├── object-keys@1.0.9 
├── UNMET PEER DEPENDENCY react@15.0.1
├── UNMET PEER DEPENDENCY react-dom@15.0.1
└── react-pdf@0.0.10 

npm WARN react-pdf@0.0.10 requires a peer of react@^0.13.3 || ^0.14 but none was installed.
npm WARN react-pdf@0.0.10 requires a peer of react-dom@^0.13.3 || ^0.14 but none was installed.
```

**Comment by daraclare** [comment]
I have the same issue, did you get it resolved? Apologies, I know this is an old post.

**Comment by chadfennell** [comment]
@daraclare There is a PR that is really close here: https://github.com/nnarhinen/react-pdf/pull/29.

**Comment by wojtekmaj** [comment]
@devang-kredx @daraclare I don't know if you are still interested, but I actively maintain a fork of this repo that fully supports React 15+. Check it out: https://github.com/wojtekmaj/react-pdf

**Comment by daraclare** [comment]
Thanks @wojtekmaj much appreciated!

**Comment by wojtekmaj** [comment]
Fixed in #34.
